### PR TITLE
Refact zepp

### PR DESCRIPTION
**Pull request by sancyx**
related issue: #7 
Fixes & enhacements to zeppelin-=spark chart:

- copy all charts into the spotguide
- use latest Spark 2.4 image
- support for namespace scoped deployment
- support usage without of external-dns

### CODE REVIEW

**Review by stoader — DISMISSED** [review]

**Review by lpuskas — DISMISSED** [review]

File: .banzaicloud/charts/spotguide-zeppelin/charts/zeppelin-spark/README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# Zeppelin-Spark Chart
+
+This is a composite chart deploying the following sub-charts:
+
+- [Zeppelin](https://github.com/banzaicloud/banzai-charts/tree/master/zeppelin)
```
Comment (baluchicken) [inline .banzaicloud/charts/spotguide-zeppelin/charts/zeppelin-spark/README.md]: Please update link to point the local chart

File: .banzaicloud/charts/spotguide-zeppelin/charts/zeppelin-spark/README.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  - [Spark Shuffle Service](https://github.com/banzaicloud/banzai-charts/tree/master/spark-shuffle) only if enabled (default *sparkShuffle.enabled=false*)
+
+## Chart Details
+
+## Installing the Chart
+
+To install the chart:
+
+```
+$ helm install banzaicloud-stable/zeppelin
```
Comment (baluchicken) [inline .banzaicloud/charts/spotguide-zeppelin/charts/zeppelin-spark/README.md]: Please specify how to add banzaicloud-stable repo to your helm

**Review by stoader — DISMISSED** [review]

File: .banzaicloud/charts/spotguide-zeppelin/charts/zeppelin-spark/README.md (hunk 1 above)
Comment (sancyx) [reply .banzaicloud/charts/spotguide-zeppelin/charts/zeppelin-spark/README.md]: will remove README files, as these are not standalone charts, they make sense only within the context of the spotguide.

File: .banzaicloud/charts/spotguide-zeppelin/charts/zeppelin-spark/README.md (hunk 2 above)
Comment (sancyx) [reply .banzaicloud/charts/spotguide-zeppelin/charts/zeppelin-spark/README.md]: will remove README files, as these are not standalone charts, they make sense only within the context of the spotguide.

**Review by lpuskas — APPROVED** [review]
lgtm

**Pull request merged**